olm-catalog/serverless-operator/project.yaml

```diff
@@ -13,11 +13,6 @@ olm:
         list:
             - stable
             - stable-1.34
```

Comment:
If the channel list will only contain the latest 2 releases, we will need to change `bumpso` tool to only have 

```
	channelsList := []interface{}{
		defaultChannel,
		fmt.Sprintf("stable-%d.%d", newVersion.Major, newVersion.Minor),
  }
```